<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegAndLogIn</class>
 <widget class="QMainWindow" name="RegAndLogIn">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegAndLogIn</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>72</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2:pressed
{
	background-color: rgb(45, 64, 71);
	border:none;
}
#pushButton_2
{
	color: white;
}
#pushButton_2:hover:!pressed
{
	 color:rgb(17,164,192);
}</string>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>187</x>
      <y>440</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3
{
	color: white;
}

#pushButton_3:pressed
{
	background-color: rgb(45, 64, 71);
	border:none;
}

#pushButton_3:hover:!pressed
{
    color:rgb(17,164,192);
}
</string>
    </property>
    <property name="text">
     <string>Can not log in?</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_4:pressed
{
	background-color: rgb(45, 64, 71);
	border:none;
}
#pushButton_4
{
	color: white;
}
#pushButton_4:hover
{
	 color:rgb(17,164,192);
}
</string>
    </property>
    <property name="text">
     <string>Log in</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>10</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#checkBox
{
	color: white;
}
</string>
    </property>
    <property name="text">
     <string>I agree with</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(17,164,192);
border-radius: 5px;
padding: 0 4px;
</string>
    </property>
    <property name="placeholderText">
     <string>Your e-mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(17,164,192);
border-radius: 5px;
padding: 0 4px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>115</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(17,164,192);
border-radius: 5px;
padding: 0 4px;
</string>
    </property>
    <property name="placeholderText">
     <string>First name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>180</y>
      <width>115</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(17,164,192);
border-radius: 5px;
padding: 0 4px;</string>
    </property>
    <property name="placeholderText">
     <string>Last name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(17,164,192);
border-radius: 5px;
padding: 0 4px;</string>
    </property>
    <property name="placeholderText">
     <string>Your e-mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(17,164,192);
border-radius: 5px;
padding: 0 4px;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>330</y>
      <width>231</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(17,164,192);
border-radius: 5px;
padding: 0 4px;
background-color:rgb(45, 64, 71);
color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>440</y>
      <width>135</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_6
{
	color: white;
}

#pushButton_6:pressed
{
	background-color: rgb(45, 64, 71);
	border:none;
}

#pushButton_6:hover:!pressed
{
    color:rgb(17,164,192);
}
</string>
    </property>
    <property name="text">
     <string>Help with registration</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>370</y>
      <width>231</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid rgb(17,164,192);
border-radius: 5px;
padding: 0 4px;
background-color:rgb(45, 64, 71);
color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>240</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Retry</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>297</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>9</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_8
{
	color: white;
}

#pushButton_8:pressed
{
	background-color: rgb(45, 64, 71);
	border:none;
}

#pushButton_8:hover:!pressed
{
    color:rgb(17,164,192);
}
</string>
    </property>
    <property name="text">
     <string>Terms</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>RegAndLogIn</receiver>
   <slot>helpIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>RegAndLogIn</receiver>
   <slot>helpReg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>RegAndLogIn</receiver>
   <slot>deal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>helpIn()</slot>
  <slot>helpReg()</slot>
  <slot>deal()</slot>
 </slots>
</ui>
